postfix/postsrsd.sls

```diff
@@ -0,0 +1,6 @@
+{% from "postfix/map.jinja" import postfix with context %}
+
+postsrsd:
```

Comment:
as an outsider who's never used this would you want to make sure they've provided the pkg name to add it on or do you expect to always install it